.github/mergify.yml

```diff
@@ -58,26 +143,307 @@ pull_request_rules:
+      - and: *is_open
+      - files=README.yaml
+      - -files=README.md
+    actions:
+      comment:
+        message: |
+          > [!IMPORTANT]
+          > `README.md` is out of date.
+          >
+
+          Rebuild the `README.md` by running `make readme` and commit the changes.
+
+          ```shell
+          make init
+          make readme
+          ```
+
+          Could you fix it @{{author}}? 🙏
+
+  - name: "ask for title"
+    conditions:
+      - and: *is_open
+      - -title~=^[0-9A-Za-z]+
+    actions:
+      comment:
+        message: |
+          > [!IMPORTANT]
+          > ## Title is necessary and should not be empty.
+          >
+          > Kindly provide a meaningful title for this Pull Request.
+
+  - name: "ask for description"
+    conditions:
+      - and: *is_open
+      - -body~=[0-9A-Za-z]{10,}
+      - body~=(Describe high-level what changed)
+    actions:
+      comment:
+        message: |
+          > [!IMPORTANT]
+          > ## Description is necessary and should not be empty.
+          >
+          > Kindly provide details with **what** was changed, **why** it was changed.
 
   - name: "remove outdated reviews"
     conditions:
+      - and: *is_open
       - or: *default_branch
     actions:
       dismiss_reviews:
         changes_requested: true
         approved: true
         message: "This Pull Request has been updated, so we're dismissing all reviews."
 
-  - name: close stale pull request
+  - name: "remove triage label if approved"
+    conditions:
+      - and: *is_open
+      - '#approved-reviews-by>=1'
+    actions:
+      label:
+        remove:
+          - triage
+
+  - name: label inactive PRs as stale
+    conditions:
+      - and: *is_open
+      - commits[*].date_committer < 30 days ago
+    actions:
+      label:
+        toggle: ["stale"]
+
+  - name: close automated PRs with persistent merge conflicts quickly
+    conditions:
+      - and: *is_open
+      - or: *automated_prs
+      - "conflict"
+      - commits[*].date_committer < 1 days ago
+    actions:
+      close:
+        message: |
+          This automated PR has been closed due to merge conflicts.
+
+  - name: close stale PRs with merge conflicts
+    conditions:
+      - and: *is_open
+      - "conflict"
+      - commits[*].date_committer < 30 days ago
+    actions:
+      close:
+        message: |
+          This PR has been closed due to inactivity and merge conflicts.
+          Please resolve the conflicts and reopen if necessary.
+
+  - name: close stale pull request after 90 days
+    conditions:
+      - and: *is_open
+      - or: *default_branch
+      - commits[*].date_committer < 90 days ago
+    actions:
+      close:
+        message: |
+          This pull request looks stale. Feel free to reopen it if you think it's a mistake.
+
+  - name: close pull request waiting on feedback for 1 month
     conditions:
+      - and: *is_open
       - or: *default_branch
-      - -closed
+      - "label~=(feedback)"
```

Comment:
Instead of using this label, could we use `#commented-reviews-by > 0` or `#changes-requested-reviews-by > 0`?

Reply:
let's do both. Good suggestion.